go/stellar/remote/interface.go

```diff
@@ -16,7 +16,7 @@ type Remoter interface {
 	AcquireAutoClaimLock(context.Context) (string, error)
 	ReleaseAutoClaimLock(context.Context, string) error
 	NextAutoClaim(context.Context) (*stellar1.AutoClaim, error)
-	RecentPayments(ctx context.Context, accountID stellar1.AccountID, limit int) (res []stellar1.PaymentSummary, err error)
+	RecentPayments(ctx context.Context, accountID stellar1.AccountID, limit int) (stellar1.PaymentsPage, error)
```

Comment:
This doesn't have page cursor param. Is that on purpose?

Reply:
It's supposed to.  I'll check.